core/benchmark/storage/src/main.rs

```diff
@@ -1704,31 +1697,29 @@ impl TxReplayer {
                 }
             }
             latest_state
-                .set::<Account>(
-                    &latest_state.account_key(&receiver),
-                    &account,
-                )
+                .set::<Account>(&latest_state.account_key(&receiver), &account)
                 .unwrap();
             self.ops_counts.set(self.ops_counts.get() + 2);
         }
 
         self.tx_counts.set(self.tx_counts.get() + 1);
         if self.tx_counts.get() % Self::EPOCH_TXS == 0 {
-            unsafe {
-                *last_state_root = Self::commit(
-                    std::ptr::read(latest_state),
-                    self.tx_counts.get(),
-                    self.ops_counts.get(),
-                );
-                std::ptr::write(
-                    latest_state,
-                    StateDb::new(
-                        self.storage_manager
-                            .get_state_at(*last_state_root)
-                            .unwrap(),
-                    ),
-                );
-            }
+            *last_epoch_id = H256::from(U256::from(&*last_epoch_id) + 1);
```

Comment:
The last_state_root is logged and used as input for testing. If there are any error in transaction processing, the state root is wrong. Please change it back to state_root.

Reply:
You may simply use the state root of the delta mpt.

Reply:
There's no public interface to get `State` from a state root hash. Should I construct `StateRootWithAuxInfo` manually?

Reply:
in the new commit only epoch id 0 is used